### DISCUSSION TITLE
[Project proposal review] Chatbot for Async API

### DISCUSSION THREAD

**Discussion by anik8das** [postmanlabs/gsoc#33] [Ideas]
Greetings! 

I'm Aniket Das, a sophomore at Minerva Schools at KGI. I have finished drafting my proposal for creating a chatbot to help the users with AsyncAPI specifications and would appreciate your review on it before submission.
Link: https://docs.google.com/document/d/1qjjMYhAL8jm4d8U0fCnkqeDmvC55Uwvntt3eohLtJTA/edit?usp=sharing

I would love feedback on what parts need additional details, what implementation can be improved, and how I can improve my overall application. Thank you for your time and consideration!

Best,
Aniket

**Comment by Sharath-Postman** [comment]
Thank you for writing in and its my understanding that your project proposal has been reviewed and you are in constant communication with the relevant mentor.

Do let me know if otherwise.

Cheers
